src/api/subscriptions.js

```diff
@@ -96,6 +96,12 @@ export class Subscriptions {
    */
   showAbbrvOffer(opt_options) {}
 
+  /**
+   * Show contribution options.
+   * @param {!OffersRequest=} opt_options
+   */
+  showContributionOptions(opt_options) {}
```

Comment:
the term "options" is really confusing. Please make it more clear what is going in here

Reply:
This shows the contribution options to the users to select from. 
Added more descriptive comments.